src/Orckestra.Composer/Services/ICurrencyConversionSettingsService.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Orckestra.Composer.Services
+{
+    public interface ICurrencyConversionSettingsService
```

Comment:
Could you please rename to ICurrencyProvider
Also use this Provider on other places, not just in mapper
Name the method like GetCurrency - not GetScopeCurrency 
We will have implementation which returns currency from scope, but other can have some other implementation